.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "prettier.singleQuote": false
```

Comment:
Is it intentional to include this? file I noticed this setting isn't needed in the CesiumJS repo.